typescript/src/2_interfaces/index.ts

```diff
@@ -40,8 +51,15 @@ export default (() => {
   // • Fix whatever is wrong with `tampa`
 
   // [do not edit] (pretend this is coming from external `foo.d.ts` lib)
+  
+  interface Coords {
+    latitude : number,
+    longitude: number,
+  }
+  
   interface City {
     name: string;
+    coords: Coords
```

Comment:
As mentioned in the instructions and comments we should not edit the City interface. Instead, we should extend the City interface.

Check how to extend interfaces.